readme.md

```diff
@@ -1,54 +1,36 @@
+Please refer  to [Dynatrace log forwarder](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd) Documentation.
 
 # Viewing Azure logs in Dynatrace UI
- Please [refer to documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#view-azure-logs).
+Please refer to [documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#view-azure-logs).
 
 # Self-monitoring
+Please refer to [self-monitoring](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#self-monitoring-optional)  Documentation.
 
-### Self-monitoring metrics
-Please [refer to documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#self-monitoring-optional).
-
-#### Azure Authentication
-Managed identity from Azure Active Directory (AD) is used to authenticate to Azure. It allows an app to easily access other Azure AD-protected resources.
-
-To use Managed identity for Function App, after deployment of azure-log-forwarder is finished you need to add a system-assigned identity. 
-In the portal you need to go to Settings section of Function App and select Identity. Within the System assigned tab, switch Status to On.
-One more step is left - in Azure role assignments you need to grant permission **Monitoring Metrics Publisher** to Resource Group where Function App is deployed.
-
-More info here: [How to use managed identities for App Service and Azure Functions](https://docs.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=dotnet)
-
-## Pricing
+# Pricing
  Ingested logs will consume DDUs. For more details: 
   - [Azure service monitoring consumption](https://www.dynatrace.com/support/help/reference/monitoring-consumption-calculation/#expand-azure-service-monitoring-consumption-103)
   - [How to calculate consumption](https://www.dynatrace.com/support/help/reference/monitoring-consumption-calculation/log-monitoring-consumption/)
 
+# Additional resources
+- [Architecture](Architecture.md)
+- ### Azure Authentication:
+   Managed identity from Azure Active Directory (AD) is used to authenticate to Azure. It allows an app to easily access other Azure AD-protected resources.
+   To use Managed identity for Function App, after deployment of azure-log-forwarder is finished you need to add a system-assigned identity. 
+   In the portal you need to go to Settings section of Function App and select Identity. Within the System assigned tab, switch Status to On.
+   One more step is left - in Azure role assignments you need to grant permission **Monitoring Metrics Publisher** to Resource Group where Function App is deployed.
+   More info here: [How to use managed identities for App Service and Azure Functions](https://docs.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=dotnet)
+- GCP [monitoring](https://github.com/dynatrace-oss/dynatrace-gcp-function)
+- AWS [monitoring](https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder)
```

Comment:
It links to log forwarder, not monitoring.

Reply:
Changed.